index.html

```diff
@@ -0,0 +1,526 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Florence</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/bootstrap/bootstrap.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:300|Poppins:300,600" rel="stylesheet">
+    <link href="./public/stylesheets/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+  <div id="main">
```

Comment:
This could have been a `<main>` instead, though the layout might ideally be:

```html
<header></header>
<main></main>
<footer></footer>
```

It looks like you're only using this as a wrapper with `overflow: hidden` so maybe something like `<div class="wrapper">`. Though I'm also wondering if you need it at all. Looks like there is some horizontal overflow issues on small screens that could be prevented with more time to investigate the culprit.